docs/dev/setup/jenkins-gitlab.rst

```diff
@@ -1208,7 +1209,7 @@ Jenkins
 .. _artemis-1:
 
 Artemis
--------
+~~~~~~~
 
 1. In ``docker-compose.yml``
```

Comment:
If we fix the local setup documentation, we could fix some possible issues in this section as well :)

1. I think Point 2.2 is obsolete and can be deleted
2. the space in "range .NetworkSettings" seems to be wrong. Correct version: "range.NetworkSettings"
3. Maybe the profile "scheduling" should be added to point 1.2

Reply:
Good points. Could you directly fix those issues?

Reply:
Done. Point 2.1 was also obsolete and would even lead to a startup failure if applied.